testing/trino-server-dev/etc/config.properties

```diff
@@ -37,6 +37,7 @@ plugin.bundles=\
   ../../plugin/trino-hive/pom.xml,\
   ../../plugin/trino-hudi/pom.xml,\
   ../../plugin/trino-example-http/pom.xml,\
+  ../../plugin/trino-faker/pom.xml,\
```

Comment:
Try running the dev server.
I think you need a catalog config in `testing/trino-server-dev/etc/catalog` as well